### DISCUSSION TITLE
Edit status from table view

### DISCUSSION THREAD

**Discussion by Jeyakumarrathnaselvi** [directus/directus#12976] [Archive]
![unknown](https://user-images.githubusercontent.com/36895131/164910943-fc5d2dcb-8b96-4c83-8d80-30dc0d2a1965.png)

Example 

- Click and change the status without going to the edit page for each item
- Modify numbers from table view when we need to edit for each items

**Comment by benhaynes** [comment]
That's the thinking with a "Spreadsheet" Layout... or adding an incline edit feature to the current Table layout. We don't have either now, but would review any PRs or it could be sponsored.
